device/droidian-boot-wlan.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 
-echo 1 > /sys/kernel/boot_wlan/boot_wlan
+[ -e /sys/kernel/boot_wlan/boot_wlan ] && echo 1 > /sys/kernel/boot_wlan/boot_wlan
+
+[ -e /dev/wlan ] echo ON > /dev/wlan
+
+[ -e /dev/wmtWifi ] echo 1 > /dev/wmtWifi
```

Comment:
Here as well

Reply:
oups. sorry for that. will fix as soon as I get home

Reply:
Should be fixed @g7